jf_agent/git/github.py

```diff
@@ -39,18 +39,12 @@ def load_and_dump(
     # Logic to help with debugging the scopes that clients have with their API key
     try:
         scopes = git_conn.get_scopes_of_api_token()
-        agent_logging.log_and_print(
-            logger,
-            logging.INFO,
+        logger.debug(
```

Comment:
Q: Why `debug` over `info` here?

Reply:
This is something I added in for the github ingestion, when I was trying to figure out what permissions people were using for the legacy (REST) API endpoint. It's not something we really need here, but since the GQL cut over is on going I decided to leave it in here for github ingestion

It's something in theory only we care about, so I switched it to a debug call